calyx/src/passes/static_timing.rs

```diff
@@ -124,7 +126,7 @@ impl Visitor for StaticTiming {
                     incr["left"] = ? fsm["out"];
                     incr["right"] = ? fsm_one["out"];
                     fsm["in"] = fsm_incr ? incr["out"];
-                    fsm["write_en"] = fsm_incr ? signal_on["out"];
+                    fsm["write_en"] = body_not_done ? signal_on["out"];
```

Comment:
Why is this change needed?

Reply:
I'm not entirely convinced that this couldn't also be the `fsm_incr` but this involved changing the fewest lines of the output code, but maybe for symmetry it should be the same since other writes seem to set the `write_en` on their own

Reply:
This should remain fsm incr.

Reply:
Makes sense, I'll make the change